utils/eth1.go

```diff
@@ -140,7 +140,13 @@ func formatAddress(address []byte, token []byte, name string, isContract bool, l
 	}
 
 	// setting tooltip & limit name/address if necessary
-	addressString := FixAddressCasing(fmt.Sprintf("0x%x", address))
+
+	addressString := ""
+	if len(address) > 22 || len(address) < 20 { // It seems this function is also used to format other hashes and not onyl 'normal' addresses
```

Comment:
Why do you allow a byte size of 21 & 22 for addresses?
The "0x" at the start would only be one byte but also cannot be/is not allowed to be part of `address`.

You can use the function `IsEth1Address` or `IsValidEth1Address` here.